lib/discourse_code_review/source/github_pr_service.rb

```diff
@@ -151,10 +151,14 @@ def pull_requests(repo_name)
       pr_querier.pull_requests(owner, name)
     end
 
-    def associated_pull_requests(repo_name, commit_sha)
+    def associated_pull_requests(repo_name, commit_sha, include_external: false)
       owner, name = repo_name.split('/', 2)
 
-      pr_querier.associated_pull_requests(owner, name, commit_sha)
+      prs = pr_querier.associated_pull_requests(owner, name, commit_sha)
+
+      return prs if include_external
```

Comment:
This reads a little strangely, though it may just be me. I would flip this:

```ruby
if !include_external
  return prs.reject { |pr| external?(pr) }
end

prs